samples/Samples.Schemas.Chat/ChatSubscriptions.cs

```diff
@@ -35,8 +34,7 @@ public ChatSubscriptions(IChat chat)
 
     private IObservable<Message> SubscribeById(IResolveFieldContext context)
     {
-        var messageContext = (MessageHandlingContext)context.UserContext;
-        var user = messageContext.Get<ClaimsPrincipal>("user");
+        var user = (ClaimsPrincipal)context.UserContext["user"];
```

Comment:
Previously the subscription code would create its own UserContext.  Now the user context is whatever is defined by the user.

Reply:
Note that previously the ClaimsPrincipal would likely always be null from a browser, because it is impossible for a browser to send the HTTP Authorize header.  I'm not even sure if it can send cookies for cookie authentication.